tests/spec/modules/modal/option-modal.spec.js

```diff
@@ -0,0 +1,77 @@
+import OptionModal from '../../../../src/modules/modal/option-modal';
+import appConst from '../../../../src/appConst.json';
+import {
+  appendHtmlText,
+  showHtml,
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Warning - 'showHtml' is defined but never used. (no-unused-vars)